src/config.ts

```diff
@@ -78,6 +78,34 @@ export const SCENE_TILE_PROVIDERS = [
   },
 ]
 
+export const TOUR = {
+  algorithm: ':first-child',
+  /*
+  apiKeyInstructions: `
+    To get an API key, ask the experts at <a
+      href="https://rocketchat.gs.mil/channel/planetdas_askanexpert"
+      target="_blank"
+    >Rocket.Chat #planetdas_askanexpert</a>.
+  `,
+  */
+  apiKeyInstructions: `
+    To get an API key you must have a Planet account.  For more information see <a
+      href="https://support.planet.com/hc/en-us/articles/212318178-What-is-my-API-key-"
```

Comment:
Same here I suppose

Reply:
This is the configuration file; this was added so that it can be configured in whatever way is needed for the given environment.